src/lib/dapp/DAppSigner.ts

```diff
@@ -63,47 +113,174 @@ export class DAppSigner implements Signer {
   }
 
   getNetwork(): { [key: string]: string | AccountId } {
-    throw new Error('Method not implemented.')
+    return this._getHederaClient().network
   }
 
   getMirrorNetwork(): string[] {
-    throw new Error('Method not implemented.')
+    return this._getHederaClient().mirrorNetwork
   }
 
   getAccountBalance(): Promise<AccountBalance> {
-    throw new Error('Method not implemented.')
+    return this.call(new AccountBalanceQuery().setAccountId(this.accountId))
```

Comment:
Note here: I believe the AccountBalanceQuery is going to be deprecated at some point, not sure what the alternative is  
![image](https://github.com/hashgraph/hedera-wallet-connect/assets/3913019/1e5d3910-aa9e-48c4-b80c-b6656b2ab04f)

Reply:
I think we may need to wait for an update to the signer interface in the hashgraph/sdk before updating this...

Reply:
AccountBalanceQuery in the SDK is leveraging mirror node to make up for the changes in consensus node query support, except for the additional latency that ma occur after a transfer transaction affecting the account being queried, it's transparent to developers.